web/js-quickstart/src/index.js

```diff
@@ -198,11 +205,15 @@ toggleScreenshare.onclick = async () => {
 function onConnection(isConnected) {
   if (isConnected) {
     form.classList.add("hide");
+    joinBtn.innerHTML = joinBtnContent;
+    joinBtn.style.opacity = 1;
+    header.classList.remove("hide");
     conference.classList.remove("hide");
     leaveBtn.classList.remove("hide");
     controls.classList.remove("hide");
   } else {
     form.classList.remove("hide");
+    header.classList.add("hide");
```

Comment:
`[header, conference, leavebtn, controls].forEach(element => element.classList.add('hide'))`

Reply:
similary remove from above can be done